public_html/wp-content/plugins/wordcamp-forms-to-drafts/wordcamp-forms-to-drafts.php

```diff
@@ -378,11 +380,19 @@ protected function get_speaker_from_user_id( $user_id ) {
 	 * @return int | WP_Error
 	 */
 	protected function create_draft_speaker( $speaker ) {
+		$content = ( ! empty( $speaker['Your Bio'] ) ) ? $speaker['Your Bio'] : '';
+
+		if ( $content ) {
+			$content = wpautop( $content );
+			$content = str_replace( '<p>', "<!-- wp:paragraph -->\n<p>", $content );
+			$content = str_replace( '</p>', "</p>\n<!-- /wp:paragraph -->", $content );
```

Comment:
Is it worth doing that conversion here? Won't G do it anyway if they save in the block editor? Or do we need to do it to avoid it being set as a Classic block?

Same applies to below code too.

Reply:
In my testing, content without the G grammar loads a Classic block, so I think this is preferable.

Reply:
👍